pkg/storage/storage.go

```diff
@@ -50,7 +50,7 @@ func (s *Storage) Get(name string, version int32) (*rspb.Release, error) {
 
 // Create creates a new storage entry holding the release. An
 // error is returned if the storage driver failed to store the
-// release, or a release with identical an key already exists.
+// release, or a release with identical a key already exists.
```

Comment:
I missed this the first time somehow. You can actually just remove the "a" to make it correct

Reply:
as you wish.already fixed

Reply:
> I missed this the first time somehow. You can actually just remove the "a" to make it correct

as you wish.already fixed